src/core/core.controller.js

```diff
@@ -953,8 +961,10 @@ export default class Chart {
 				}
 			};
 
-			me.platform.addEventListener(me, 'resize', listener);
+			me.attached = me.platform.addEventListener(me, 'resize', listener);
```

Comment:
Is this a breaking API change that we need to document now?

Reply:
I don't think a new return value is breaking, but it could be documented for sure.
Its not something you'd expect `addEventListener` to return though, that's why I didn't bother to document it.

Maybe the logic should be moved to core from platform:
```js
if (platform.isAttached(me)) {
  platform.addEventListener(me, 'resize', ...)
  platform.addEventListener(me, 'detach', ...)
} else {
  platrofm.addEventListener(me, 'attach', ...)
}
```

Reply:
I think this would be a good strategy. It would allow for alternate implementations without returning from `addEventListener`